test/rules.jl

```diff
@@ -1,3 +1,6 @@
+# Define these in own module so can rerun test and new instances will be defined if we re`include` this file
```

Comment:
did you mean to have a `module` here?

Reply:
Nope this line should have been deleted.
I was trying that but it didn't work

Reply:
```suggestion
```